src/brijesh/Mall.java

```diff
@@ -0,0 +1,66 @@
+package brijesh;
+
+public class Mall {
+	int numberOfItems;
+	String name;
+	String paymentMode;
+	boolean requiredBill;
+	boolean haveCarryBag;
+	String section;
+	
+	void itemAddedToCart(int items) {
+		this.numberOfItems=items;
+	}
+	
+	void customerName(String name) {
+		this.name=name;
+	}
+	
+	void paymentDoneUsing(String paymentMode) {
+		this.paymentMode=paymentMode;	
+	}
+	
+	void displayCutomerInfo(Mall mall) {
+		mall.decideSection();
```

Comment:
Here, you can directly call the decideSection() method.
this is also correct. 
you should call decideSection()  from the main method not from any internal method() bit different than requirement.
Refer: https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/pull/6/files